src/VisualStudio/CSharp/Test/ProjectSystemShim/LegacyProject/CSharpCompilerOptionsTests.cs

```diff
@@ -114,32 +114,28 @@ public void ProjectOutputPathAndOutputExeNameChange()
             project.SetOutputFileName(initialPath);
             Assert.Equal(initialPath, project.GetOutputFileName());
 
-            var outputs = (CompilationOutputFilesWithImplicitPdbPath)environment.Workspace.GetCompilationOutputs(project.Test_VisualStudioProject.Id);
-            Assert.Equal(initialPath, outputs.AssemblyFilePath);
+            Assert.Equal(initialPath, project.Test_VisualStudioProject.IntermediateOutputFilePath);
```

Comment:
Note the tests here changed in a subtle way: now this is asserting the underlying value of the IntermediateOutputFilePath on VisualStudioProject, but nothing is actually asserting it's getting to the project snapshot itself, I don't think?